{"name": "Pandemia_di_COVID-19_in_Asia", "id": 8802536, "predecessors": [{"name": "Storia_contemporanea_asiatica", "id": 88402}, {"name": "Categorie", "id": 41474}], "successors": []}